pkgs/tools/misc/fdupes/default.nix

```diff
@@ -1,14 +1,14 @@
-{stdenv, fetchurl}:
+{stdenv, fetchFromGitHub}:
 
-stdenv.mkDerivation {
-  name = "fdupes-1.51";
-  src = fetchurl {
-    url = https://github.com/adrianlopezroche/fdupes/archive/fdupes-1.51.tar.gz;
-    sha256 = "11j96vxl9vg3jsnxqxskrv3gad6dh7hz2zpyc8n31xzyxka1c7kn";
-  };
+stdenv.mkDerivation rec {
+  name = "fdupes-git";
```

Comment:
Please remove `-git` and add `-VERSION` where version is the date of the commit (I mean `src.rev`).